packages/trip-details/src/index.js

```diff
@@ -192,6 +214,46 @@ function TripDetails({
             }
           />
         )}
+        {containsFlex && (
+          <TripDetail
+            icon={<Route size={17} />}
+            summary="This trip includes flexible routes."
+          />
+        )}
+        {pickupBookingInfo &&
+          pickupBookingInfo.map(info => (
+            <TripDetail
+              key={info.pickupMessage}
```

Comment:
Sort props.

Reply:
Sorted in 5d40040651a55901e69f8a04401a657153658723